libs/shared/src/consts/providers/provider-credentials.ts

```diff
@@ -5,7 +5,7 @@ const mailConfigBase: IConfigCredentials[] = [
   {
     key: CredentialsKeyEnum.From,
     displayName: 'From email address',
-    description: 'Use the authenticated email address from the delivery provider you will send emails from.',
+    description: 'Use the same email address you used to login to your email provider',
```

Comment:
```suggestion
    description: 'Use the same email address you used to authenticate your delivery providerr',
```
@oba2311 how about that?